lib/opal/parser/lexer.rb

```diff
@@ -73,6 +76,26 @@ def initialize(source, file)
       @start_of_lambda = nil
     end
 
+    # Set the default prefix to use for direct javascript method calls.
+    # By default, direct javascript method calls are not supported by
+    # the lexer.  If true is given, a the ".JS." token is used for
+    # direct javascript method calls.  If false or nil is given, direct
+    # javascript method calls are not supported.  If a String is given,
+    # that string preceeded by a "." is used for direct javascript
+    # method calls.
+    #
+    # @param prefix [String|true|false|nil] prefix to use
+    def js_prefix=(prefix)
+      @js_prefix = case prefix
+                   when true
+                     DEFAULT_JS_PREFIX
+                   when false, nil
+                     nil
+                   else
+                     /\.#{Regexp.escape(prefix)}/
```

Comment:
This is by design, to allow the user to choose a prefix that doesn't end in a `.`.  Would you like to force a `.` at the end?